components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -85,10 +86,26 @@ module micro_p3
      real(rtype), dimension(:), pointer :: p
   end type realptr
 
+  interface var_coef
+    module procedure var_coef_rtype
+  end interface var_coef
+
 contains
 
   !==================================================================================================!
+  elemental function var_coef_rtype(relvar, expon) result(res)
+
+  ! Finds a coefficient for process rates based on the relative variance
+  ! of cloud water.
+  real(rtype), intent(in) :: relvar
+  real(rtype), intent(in) :: expon
+  real(rtype) :: res,res_tmp
 
+  res_tmp = gamma(relvar+expon)/gamma(relvar)
+  res = res_tmp/relvar**expon
```

Comment:
This should probably use `bfb_pow` to handle the exponent.  Those are the set of macros that Jim set up to handle basic math operations and remain BFB between C++ and Fortran.

Reply:
The same goes for `bfb_gamma` for the use of the gamma function.

Reply:
good catch, Aaron! I agree.